generateRandomId.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
+const writer = csvWriter();
+const faker = require('faker');
+
+
+const generateId = () => {
+  writer.pipe(fs.createWriteStream('id.csv'));
+  for (let i = 0; i < 10000; i++) {
+    writer.write({
+      id: faker.random.number({'min': 1,'max': 10000})
+    });
```

Comment:
I think we may need to change the number of random Id's to 250 depending on what kind of feedback we get about the max limit of event types and why we're dropping data points